Bio/PDB/MMCIFParser.py

```diff
@@ -122,6 +165,7 @@ def _build_structure(self, structure_id):
         structure_builder = self._structure_builder
         structure_builder.init_structure(structure_id)
         structure_builder.init_seg(" ")
+        structure_builder.set_header(self._get_mmcif_header_dict(mmcif_dict))
```

Comment:
The `PDBParser` class implements a `get_header` function that handles this step. To be coherent with that implementation, I'd do the same. The other reason being that the header is a separate piece of (meta)data than the coordinates so logically the two parsing steps should be clearly delineated and separated.

Reply:
I think this is done, if I understand your intent correctly.